readme.md

```diff
@@ -224,11 +227,12 @@ into markdown and saved to readme.md.
 
 ## Changelog ##
 
-### 2.4.0 (202x-xx-xx) ###
+### 2.4.0 (2024-xx-xx) ###
```

Comment:
Should this date be filled in before deployment?

Reply:
Before release, yes, but not before merge. I always set the date before I release the version.